doc/classes/RenderingDevice.xml

```diff
@@ -487,6 +493,31 @@
 				Returns the index of the last frame rendered that has rendering timestamps available for querying.
 			</description>
 		</method>
+		<method name="get_device_allocation_count" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns how many allocations the GPU has performed for internal driver structures.
+				This is only used by Vulkan in Debug builds and can return 0 when this information is not tracked or unknown.
+			</description>
+		</method>
+		<method name="get_device_allocs_by_object_type" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="type" type="int" />
+			<description>
+				Same as get_device_allocation_count but filtered for a given object type.
+				The type argument must be in range [code]【0; get_tracked_object_type_count)[/code]. If get_tracked_object_type_count is 0, then type argument is ignored and always returns 0.
```

Comment:
```suggestion
				The type argument must be in range [code][0, get_tracked_object_type_count)[/code]. If [method get_tracked_object_type_count] is 0, then type argument is ignored and always returns 0.
```

Reply:
**Small warning:** I cannot change `【0; get_tracked_object_type_count)` to `[0; get_tracked_object_type_count)` because it fails pre commit checks with:

```
ERROR: RenderingDevice.xml: Tag depth mismatch: too many (or too few) open/close tags in method "get_driver_allocs_by_object_type" description.
```

I tried:
 - \\[
 - \`[\`
 - \`\\[\`

The error appears whenever the description has includes some other special tag like `[method]` or `[code]`.

If you have a solution I'll gladly use it; but meanwhile I had to workaround it by using the Unicode similar character 【

Reply:
Ah then it's an escape, but I'd say since it's an integer here you should instead do:
```suggestion
				The type argument must be in range [code][0, get_tracked_object_type_count - 1][/code]. If [method get_tracked_object_type_count] is 0, then type argument is ignored and always returns 0.
```

Reply:
What a simple solution.
I hate it. And I love it.

While I prefer `[0, get_tracked_object_type_count)` notation, I was unhappy that `【` breaks text2voice for visually impaired. Your solution it is.